app/controllers/local_groups_controller.rb

```diff
@@ -53,6 +53,15 @@ def destroy
 
   private
 
+  def local_group_params
```

Comment:
I moved `local_group_params` to both `LocalGroup::CreateService` and `LocalGroup::UpdateService`. You could remove this one and add `:welcome_email_body` in services.